bootstrap/src/telemetry/otlp.rs

```diff
@@ -68,22 +74,26 @@ pub fn initialize(
 	origin: String,
 	network: String,
 ) -> Result<Metrics, Error> {
-	let export_config = ExportConfig {
-		endpoint,
-		timeout: Duration::from_secs(10),
-		protocol: Protocol::Grpc,
-	};
-	let provider = opentelemetry_otlp::new_pipeline()
-		.metrics(opentelemetry_sdk::runtime::Tokio)
-		.with_exporter(
-			opentelemetry_otlp::new_exporter()
-				.tonic()
-				.with_export_config(export_config),
-		)
-		.with_period(Duration::from_secs(10))
-		.with_timeout(Duration::from_secs(15))
+	let exporter = MetricExporter::builder()
+		.with_tonic()
+		.with_endpoint(&endpoint)
+		.with_protocol(Protocol::Grpc)
+		.with_timeout(Duration::from_secs(10))
```

Comment:
Maybe we should think about increasing the default timeout to 30s?

Reply:
yes will increase it